src/calendar/actions/calendar/check-calendar-location.ts

```diff
@@ -0,0 +1,28 @@
+import { Fetcher } from '../../../common/api';
+import { endpoints } from '../../api/calendar/endpoints';
+import { ICheckLocationInputDto } from '../../types';
+import { errorFactory, CalendarErr } from '../../errors/errors';
+
+export const checkLocationForCalendar = async (
```

Comment:
De manière générale, il serait intéressant de reprendre la description faite des méthodes sur la doc afin de l'ajouter en commentaire ici (ou sur le provider, à voir ce qui permet d'afficher l'information à l'utilisateur à l'usage).

Par exemple :

```suggestion
/**
   * Check availabalities for given location and return calendar.
   */
export const checkLocationForCalendar = async (
```

Reply:
J'ai ajouté la doc au niveau des appels et au niveau du provider.